code/src/index.css

```diff
@@ -5,9 +5,157 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-image: linear-gradient(
+      to bottom,
+      rgba(0, 0, 0, 0.3),
+      rgba(0, 0, 0, 0.4)
+    ),
+    url("./background-image.jpg");
+  background-repeat: repeat;
+  background-attachment: fixed;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.send-thought {
+  border: 1px solid black;
+  width: 90%;
+  height: 135px;
+  margin: 30px 0;
+  padding: 10px;
+  box-shadow: 5px 5px 1px black;
+  background-color: rgb(240, 236, 236);
+  position: relative;
```

Comment:
This can be removed, together with position absolut on button and character, you would then have to wrap button and character in a div and display them as flex.